classes/Webhook/WebhookAvailability.php

```diff
@@ -46,9 +46,6 @@ public function check()
         $response = new Response();
         $curl = curl_init((new WebhookHandlerUrl())->getCheckAvailabilityUrl());
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
-        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, true);
-        curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 2);
-        curl_setopt($curl, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_2);
```

Comment:
To check the TLS version, force TLSv1.2 is mandatory in this case.
I don't understand why we have 2 classes to check the TLS. Can we have only one ?

Reply:
Added